cfme/tests/automate/test_automate_manual.py

```diff
@@ -970,3 +927,62 @@ def test_user_requester_for_lifecycle_provision():
          1671563
     """
     pass
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+def test_remove_openshift_deployment_in_automate():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.11
+        casecomponent: Automate
+        tags: automate
```

Comment:
I would expect `testSteps` and `expectedResults` here, but I see this is for 5.11. Do you know how you would go about testing this, at least high level? If not, I think we will have to wait before we merge this. We do want to have some specifics on how to test this before it gets merged.